test/e2e/framework.go

```diff
@@ -815,3 +803,24 @@ func (data *TestData) deleteService(nsName string, svcName string) error {
 func (data *TestData) useWCPSetup() bool {
 	return data.vcClient != nil
 }
+
+func checkTrafficByCurl(ns, podname, containername, ip string, port int32, interval, timeout time.Duration) error {
```

Comment:
Does it work if we call this function precreated_vpc_test.go when validating the LB traffic?

Reply:
Yes, you can refactor it after it checked in.